torch/_dynamo/symbolic_convert.py

```diff
@@ -712,7 +708,10 @@ def step(self):
         )
 
     def run_ctx_mgr(self):
-        return contextlib.nullcontext()
+        # NB: Don't push the top level frame summary; set_current_loc will
+        # take care of it.  However, DO make sure we attach real_stack to
+        # exceptions
+        return TracingContext.current_frame(None)
```

Comment:
I don't think it is possible to exercise this case, but it's safest to make sure we set up the try...except handler.